frontend/src/types/UserTypes.ts

```diff
@@ -9,7 +9,6 @@ export type User = {
 };
 
 export type UserLabel = {
-  id: number;
```

Comment:
I think we'll need to revert this, as above, `getCurUserSelectOption` still returns an object with an `id` field, and feeds this into the state prop which takes in a `UserLabel`, so we'd have misaligned types here

Reply:
i just removed the returned id from `getCurUserSelectOption`